### DISCUSSION TITLE
Public instance

### DISCUSSION THREAD

**Discussion by comfuzio** [benbusby/whoogle-search#930] [Show and tell]
Hello everybody :) 
I made one at https://whoogle.gr/ it is behind npm and maybe will switch behind cloudflare proxy if traffic becomes too heavy.

Thank you

**Comment by Glitch3dPenguin** [comment]
This is down

**Reply by comfuzio** [reply]
Yeap. I am moving servers to new datacenter and might be down for a few more days. Sorry about that

**Reply by comfuzio** [reply]
It is up right now. Please let me know if you face any issues cause I just booted it!

**Reply by Glitch3dPenguin** [reply]
I can confirm that is is working! Thanks!

**Reply by comfuzio** [reply]
Thank you and sorry for the inconvenience :)
Now it should be in HA mode unless, I do updates to the VM no reason for it to go down :D
